commands.go

```diff
@@ -29,10 +29,6 @@ type AppendEntriesResponse struct {
 
 	// We may not succeed if we have a conflicting entry
 	Success bool
-
-	// There are scenarios where this request didn't succeed
-	// but there's no need to wait/back-off the next attempt.
-	NoRetryBackoff bool
```

Comment:
Yeah this is an interesting one. The types of things where we didn't set this to true were were kind of a miscellaneous grab of application errors like not being able to write to the log. I'd probably rather see the back pressure done as a delayed response (such as if we did retries writing to the log, etc.) vs. an immediate response asking for a back off. Removing this and keeping it for comm errors seems like the way to go.